vividus-engine/src/test/java/org/vividus/steps/ExpressionAdapterTests.java

```diff
@@ -116,14 +116,18 @@ void testSupportedExpressionNestedExpr()
     }
 
     @SuppressWarnings("unchecked")
-    @Test
-    void testNestedExpressionWithoutParams()
+    @ParameterizedTest
+    @CsvSource({
```

Comment:
use `@ValueSource` for list of values (`@CsvSource` is used for 2-dimensions data aka tables)